docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -409,27 +409,18 @@ Post-Release Review Process
 
      XXX: This step should be automated. See `FLOC-1039 <https://clusterhq.atlassian.net/browse/FLOC-1039>`_.
 
-   * The node package (``flocker-node``) should be installed on all supported platforms.
-     You can use vagrant to boot a clean Fedora 20 machine as follows:
+   * The node package (``flocker-node``) should be installed and tested on all supported platforms.
 
-     .. code-block:: console
-
-        mkdir /tmp/fedora20
-        cd /tmp/fedora20
-        vagrant init clusterhq/fedora20-updated
-        vagrant up
-        vagrant ssh
+     The acceptance tests can be run automatically using the ``run-acceptance-tests`` tool.
+     See :ref:acceptance-testing for details of the configuration files required for each platform.
 
-     Follow the :ref:`Flocker node installation documentation<installing-flocker-node>`.
+     .. code-block:: console
 
-     XXX: These steps should be automated. See (
-     `FLOC-965 <https://clusterhq.atlassian.net/browse/FLOC-965>`_,
-     `FLOC-957 <https://clusterhq.atlassian.net/browse/FLOC-957>`_,
-     `FLOC-958 <https://clusterhq.atlassian.net/browse/FLOC-958>`_
-     ).
+        $ ./admin/run-acceptance-tests --distribution=fedora-20 --provider=vagrant --flocker-version=0.3.2-1109-gd71621f --branch=digitalocean-acceptance-FLOC-958
```

Comment:
Well actually I think the version and branch options can be omitted altogether in a real release, because we'll be running the tool from the release branch which will have been tagged, so the version can be discovered automatically.